src/zcl_abapgit_object_scvi.clas.abap

```diff
@@ -0,0 +1,190 @@
+CLASS zcl_abapgit_object_scvi DEFINITION
+  PUBLIC
+  INHERITING FROM zcl_abapgit_objects_super
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    INTERFACES zif_abapgit_object .
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+
+    TYPES:
+      BEGIN OF ty_screen_variant,
+        shdsvci    TYPE shdsvci,
+        shdsvtxci  TYPE STANDARD TABLE OF shdsvtxci  WITH DEFAULT KEY,
+        shdsvfvci  TYPE STANDARD TABLE OF shdsvfvci  WITH DEFAULT KEY,
+        shdguixt   TYPE STANDARD TABLE OF shdguixt   WITH DEFAULT KEY,
+        shdgxtcode TYPE STANDARD TABLE OF shdgxtcode WITH DEFAULT KEY,
+      END OF ty_screen_variant .
+ENDCLASS.
+
+
+
+CLASS ZCL_ABAPGIT_OBJECT_SCVI IMPLEMENTATION.
+
+
+  METHOD zif_abapgit_object~changed_by.
+
+    DATA: lv_screen_variant TYPE scvariant.
+
+    lv_screen_variant = ms_item-obj_name.
+
+    SELECT SINGLE chuser
+    FROM shdsvci
+    INTO rv_user
+    WHERE scvariant = lv_screen_variant.
+    IF sy-subrc <> 0
+    OR rv_user IS INITIAL.
+      rv_user = c_user_unknown.
+    ENDIF. " IF sy-subrc <> 0
```

Comment:
please remove the end of line comments with ABAP code, in this example `" IF sy-subrc <> 0`

https://github.com/SAP/styleguides/blob/master/clean-abap/CleanABAP.md#dont-add-method-signature-and-end-of-comments